dev/docker/building/dev-env.Dockerfile

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+FROM openjdk:8-jdk-buster
+LABEL maintainer="support@datastrato.com"
+
+WORKDIR /root/gravitino
+
+COPY Dockerfile /root/gravitino
```

Comment:
I still can't understand what the purpose of this operation is.

Reply:
Me too; should it clone the github repository to the container local disk, or copy from the developer's local disk to the contianer? 

Is there any other dependencies which need be installed, such as nodejs? If no other dependency, seems it is quite simple, maybe not that necessary to provide such a docker image. What do you think?

Reply:
> Me too; should it clone the github repository to the container local disk, or copy from the developer's local disk to the contianer?
> 
> Is there any other dependencies which need be installed, such as nodejs? If no other dependency, seems it is quite simple, maybe not that necessary to provide such a docker image. What do you think?

You are correct; my Dockerfile is quite incomplete. My original intention was to provide an image that includes all the environments and third-party dependencies required for packaging with the ./gradlew build command, such as JDK, Node.js, and various other third-party dependencies. I believe you are better suited to do this because you have a clearer understanding of the dependencies required by Gravitino than I do.